backend/src/apiserver/resource/client_manager_fake.go

```diff
@@ -63,7 +63,7 @@ func NewFakeClientManager(time util.TimeInterface, uuid util.UUIDGeneratorInterf
 		experimentStore:             storage.NewExperimentStore(db, time, uuid),
 		pipelineStore:               storage.NewPipelineStore(db, time, uuid),
 		jobStore:                    storage.NewJobStore(db, time),
-		runStore:                    storage.NewRunStore(db, time),
+		runStore:                    storage.NewRunStore(db, time, nil),
```

Comment:
Should we have a fake for the third term? (I am guessing it is the metadata store)

Reply:
Could we also have an issue? I would suggest we address this as a high priority as not having the SQLLite implementation will decrease development velocity: errors will only be caught at the time of integration testing.

Reply:
I will move this out of API server. In the meantime, filed issue #907